internal/credentials/credentials_test.go

```diff
@@ -26,20 +28,33 @@ var fileCred = Credential{
 	Name:   "connect.localtest.me",
 }
 
-func credentialsFilePath(afs afero.Fs) (util.AbsolutePath, error) {
+type CredentialsTestSuite struct {
+	utiltest.Suite
```

Comment:
Updated this tests to be a testify suite for consistency with our other test files. Most changes in this file are just swapping `assert` calls to a `Suite s.*()` method. New relevant changes assert calls to a mock logger instance.